app/api/schema/attendees.py

```diff
@@ -50,7 +50,6 @@ def validate_json(self, data, original_data):
     gender = fields.Str(allow_none=True)
     birth_date = fields.DateTime(allow_none=True)
 
-    ticket_id = fields.Str(allow_none=True)
```

Comment:
Undo this and run the tests.

Reply:
@iamareebjamal I think the tests are failing due to this.

Reply:
I have tried it. The issue still persists.